### PR TITLE
added missing quote

### PR DESCRIPTION
**Pull request by aboutte**

### CODE REVIEW

File: docs/resources/xml.md.erb [hunk 1]
```diff
@@ -28,7 +28,7 @@ This file can be queried using:
     describe xml('/path/to/name.xml') do
       its('root/name') { should eq ['hello'] }
       its('root/meta/creator') { should eq ['John Doe'] }
-      its('root/array[2]/element]) { should eq ['two'] }
+      its('root/array[2]/element]') { should eq ['two'] }
```
Comment (jerryaldrichiii) [inline docs/resources/xml.md.erb]: I think you'll want to remove the `]` as well.

**Comment by jerryaldrichiii** [comment]
Great catch @aboutte!

**Review by jerryaldrichiii — APPROVED** [review]
Thanks for your first contribution to InSpec @aboutte :smile:

**Review by adamleff — APPROVED** [review]
Thanks a lot for helping us keep our documentation accurate, @aboutte :)

**Pull request merged**